website/docs/Basic_operations/delete.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 5
+---
+
+
+# Delete operation
+
+The delete operation removes a document from the database when a given query is met.
+Two methods for deleting documents in a collection include `deleteOne()` and `deleteMany()`.
+
+## Delete a single document
+
+The `deleteOne()` method removes a single document (the first document that matches the query parameter) completely from the collection.
+
+```sh
+db.collection.deleteOne({<query_params>})
+```
+
+Insert the following list of documents:
+
+```sh
+db.scientists.insertMany([{firstname: "Thomas", lastname: "Edison", born: 1847, invention: "LightBulb", nobel:true},{firstname: "Graham", lastname: "Bell", born: 1847, invention: "telephone", nobel:false},{firstname: "Nikola", lastname: "Tesla", born: 1856, invention: "Tesla coil", nobel:false}, {firstname: "Ada", lastname: "Lovelace", born: 1815, invention: "Computer programming", nobel:false}])
+```
+
+Next, delete a document from the collection where the field `nobel` is set to false.
+
+```sh
+db.scientists.deleteOne({nobel:false})
+```
+
```

Comment:
Should we write something about the response (like we did for `insert`)? And explain that only one document was deleted and what's left in the collection?